roles/infrastructure/krb5_server/tasks/freeipa.yml

```diff
@@ -30,7 +30,6 @@
 
 - name: Create FreeIPA DNS records for PVC ECS
   when:
-    - pvc_type is defined
```

Comment:
I would expect this to fail if `pvc_type` is not in the user definition or role defaults.